### ISSUE TITLE
join: [...] syntax

### ISSUE THREAD

**Issue by anikaks** [looker-open-source/malloy#140]
Currently joins work when I write `join:` on each line. 

Should `join: [...]` work?

**Comment by mtoy-googly-moogly** [comment]
yes

```
join: [ oneThing on oneKey, twoThing on twoKey ]
```

should totally work

**Comment by mtoy-googly-moogly** [comment]
I have no tests, so I have no idea if it does work.

**Comment by anikaks** [comment]
I tested it and it's not working for me currently, here's what I tried. Using the ecommerce model:
```
 join: [
    users on user_id,
    ii on inventory_item_id,
    user_order_facts on user_id
  ]
```

I get an extraneous input error on the bracket. It does actually seem to be accepting this, and queries requiring the join are running properly:
```
join: {
    users on user_id
    ii on inventory_item_id
    user_order_facts on user_id
}
```

**Comment by mtoy-googly-moogly** [comment]
doh ... yeah it should be bracket. fixing that now.

Fixed in 67712100e9ac15d90638909254809d2fa3d63b0a
